src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -38,6 +38,11 @@ public class AzureAppConfigurationOptions
         // Since multiple prefixes could start with the same characters, we need to trim the longest prefix first.
         private SortedSet<string> _keyPrefixes = new SortedSet<string>(Comparer<string>.Create((k1, k2) => -string.Compare(k1, k2, StringComparison.OrdinalIgnoreCase)));
 
+        /// <summary>
+        /// Flag to indicate whether failover to replicas automatically.
+        /// </summary>
+        public bool IsAutoFailover { get; set; }
```

Comment:
This is my first implementation of supporting this flag.  After rethinking this, I agree we should not expose this option in such a way that does not comply with the convention.

Determine `isAutoFailover` in Conenct method does make sense because it's kind of an option that closely related to the conenction.